docs/pages/blog/material-ui-v6-sneak-peek.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Material UI v6 Sneak Peek
```

Comment:
Maybe this change better communicates that v6 is close in time.

```suggestion
title: Material UI v6 now in beta
```

Reply:
Changed to stable release. I will let @samuelsycamore casts his magic for the title.